--- Python-3.8.8/configure.ac	2021-02-19 18:28:41.000000000 +0800
+++ mod/configure.ac	2023-05-25 22:05:40.922231534 +0800
@@ -1790,7 +1790,7 @@
 
     if test $ac_cv_enable_implicit_function_declaration_error = yes
     then
-      CFLAGS_NODIST="$CFLAGS_NODIST -Werror=implicit-function-declaration"
+      CFLAGS_NODIST="$CFLAGS_NODIST"
     fi
 
     # if using gcc on alpha, use -mieee to get (near) full IEEE 754
